apps/betterangels-backend/accounts/managers.py

```diff
@@ -36,7 +35,7 @@ def create_superuser(self, email: str, password: str = "", **extra_fields: Any)
             raise ValueError(_("Superuser must have is_staff=True."))
         if extra_fields.get("is_superuser") is not True:
             raise ValueError(_("Superuser must have is_superuser=True."))
-        return self.create_user(email, password, **extra_fields)
```

Comment:
What is replacing email? Username?


https://github.com/django/django/blob/89557d4c66b469616fc3a16ba11b2999233efa8d/django/contrib/auth/models.py#L159

Reply:
yup `username` will now be required